generated/app/BryzekApidocSpecV0Client.scala

```diff
@@ -15,10 +15,26 @@ package com.bryzek.apidoc.spec.v0.models {
     key: String
   )
 
+  /**
+   * Represents an additional attribute that is attached to one of the objects in
+   * apidoc. The main use case is to capture additional metadata that doesn't
+   * necessarily define the API but aids in code generation. Examples would be hints
+   * for certain code generators about classes to extend, interfaces to implement,
+   * annotations to add, names to assign to certain methods, etc. The specific
+   * attributes will be applicable only in the context of the specific code
+   * generators usings them.
+   */
+  case class Attribute(
+    name: String,
+    value: _root_.play.api.libs.json.JsObject,
+    description: _root_.scala.Option[String] = None
```

Comment:
Seeing this now - we're going to want to add the deprecation object to the attribute for consistency.